### PR TITLE
Patch documentation

### PR DESCRIPTION
**Pull request by alesanter**

### CODE REVIEW

**Comment by blackyblack** [comment]
Try to use auto-generated documentation when possible.

File: symplyphysics/laws/condensed_matter/effective_mass_of_electron_via_energy.py [hunk 1]
```diff
@@ -28,15 +30,9 @@
     validate_output,
 )
 
-effective_mass = Symbol("mass", units.mass)
+effective_mass = clone_symbol(symbols.basic.mass, "effective_mass", display_symbol="m_eff", display_latex=r"m_\text{eff}")
```
Comment (blackyblack) [inline symplyphysics/laws/condensed_matter/effective_mass_of_electron_via_energy.py]: Remove "effective_mass" text.
Use `\\` in latex and avoid r-strings.

Comment (alesanter) [reply symplyphysics/laws/condensed_matter/effective_mass_of_electron_via_energy.py]: Why not r-strings?

Comment (blackyblack) [reply symplyphysics/laws/condensed_matter/effective_mass_of_electron_via_energy.py]: I don't like em.

Comment (alesanter) [reply symplyphysics/laws/condensed_matter/effective_mass_of_electron_via_energy.py]: To clarify, you only mean r-strings in this context or everywhere?

File: symplyphysics/laws/electricity/circuits/resistance_in_serial_connection.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
+"""
+Resistivity of serial resistors
+===============================
+
+The total resistance of the circuit whose components are connected in series is the sum
+of the resistances of individual components.
+
+**Conditions:**
+
+#. The circuit is a DC one or the reactance is zero.
```
Comment (blackyblack) [inline symplyphysics/laws/electricity/circuits/resistance_in_serial_connection.py:10]: Maybe decipher `DC`? Also it is characterstic of power source, not circuit.

File: symplyphysics/laws/condensed_matter/effective_mass_of_electron_via_energy.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Wavenumber.
+
+Symbol:
+    :code:`k`
+"""
+
+law = Eq(effective_mass, units.hbar**2 / Derivative(energy(wavenumber),
+    (wavenumber, 2)))
+r"""
+:code:`m_eff = hbar^2 * Derivative(E(k), (k, 2))^(-1)`
```
[the remark this answers is not in the document]
Comment (alesanter) [reply symplyphysics/laws/condensed_matter/effective_mass_of_electron_via_energy.py:57]: I just don't want the latex version to be a big fraction.

File: symplyphysics/laws/electricity/circuits/resistance_in_serial_connection.py (hunk 2 above)
Comment (alesanter) [reply symplyphysics/laws/electricity/circuits/resistance_in_serial_connection.py:10]: No, it is a standard definition:

> A **direct current circuit** is an electrical circuit that consists of any combination of constant voltage sources, constant current sources, and resistors.

Comment (alesanter) [reply symplyphysics/laws/electricity/circuits/resistance_in_serial_connection.py:10]: But I'll remove the abbreviation.

File: symplyphysics/laws/condensed_matter/effective_mass_of_electron_via_energy.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (blackyblack) [reply symplyphysics/laws/condensed_matter/effective_mass_of_electron_via_energy.py:57]: But it is code representation. They may be different with latex.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**